### PR TITLE
Avoid unneccessary minting of private coin that will be overwritten

### PR DESCRIPTION
**Pull request by jonspock**
…ritten

Performance improvement to avoid unnecessary extra minting of zPIV

### CODE REVIEW

**Comment by furszy** [comment]
Tested ACK.

**Comment by Warrows** [comment]
@jonspock the travis build is failing because of the trailing whitespace. You should remove it.

File: src/wallet.cpp [hunk 1]
```diff
@@ -4589,7 +4589,7 @@ bool CWallet::MintToTxIn(CZerocoinMint zerocoinSelected, int nSecurityLevel, con
     }
 
     // Construct the CoinSpend object. This acts like a signature on the transaction.
-    libzerocoin::PrivateCoin privateCoin(paramsCoin, denomination);
+    libzerocoin::PrivateCoin privateCoin(paramsCoin, denomination, false);
```
Comment (Fuzzbawls) [inline src/wallet.cpp]: unnecessary trailing whitespace, needs fixing.

**Comment by blondfrogs** [comment]
I haven't seen @jonspock respond to anyone on this PR so I made the same change that didn't include the additional white space on PR #690.

**Comment by jonspock** [comment]
Just made the change. Didn't think there was a rush since this PR sat for 2 months

**Comment by Warrows** [comment]
Indeed, no rush IMO. Could you squash the two commits into one with a message under the limit please?

**Comment by jonspock** [comment]
Just go ahead with your own PR @Warrows . Don't have time for all of these nits

**Comment by Warrows** [comment]
Alright, I guess we'll just go with #690 then. Thanks for your help anyway, it's appreciated.

**Pull request closed without merging**